calico-enterprise_versioned_docs/version-3.15/_includes/release-notes/_v3.15.3-release-notes.mdx

```diff
@@ -0,0 +1,12 @@
+08 May 2023
+
+### Improvements
+
+Egress gateways will attempt to auto detect the iptables backend, preferring `nft` if available.
```

Comment:
- Update rbac.authorization.k8s.io apiVersion to v1

Reply:
Added.